base_layer/core/src/blocks/genesis_block.rs

```diff
@@ -63,6 +64,99 @@ pub fn get_genesis_block(network: Network) -> ChainBlock {
     }
 }
 
+pub fn get_stagenet_genesis_block() -> ChainBlock {
```

Comment:
Please add a test for this at the bottom of this file like we do for esme and igor.

Reply:
Added test, which found errors, that is now fixed. 👍🏻